### ISSUE TITLE
Additional breakpoints for columnset

### ISSUE THREAD

**Issue by operatorone** [madeyourday/contao-rocksolid-columns#19]
Add in the option of the column wrapper additional fields for two more breakpoints. The result might look like this:

label | class | sample size (px)
------------ | ------------- | -------------
Desktop | xx-large | ≥ 1440 px
Laptop | x-large | < 1440px
Tablet landscape | large | < 1200px
Tablet portrait | medium | < 800px
Phone landscape | small | < 640px
Phone | x-small | < 480px

The resolutions are just an example. They can be adjusted anyway. But the count of at least five breakpoints is essential. Better to have six breakpoints, so there is more flexibility.

https://github.com/madeyourday/contao-rocksolid-columns/blob/develop-v3/src/Resources/contao/dca/tl_content.php

Thanks a lot.

**Comment by operatorone** [comment]
Hi, any news for this enhancement? 😉

**Comment by ausi** [comment]
What would you think about adding a bundle configuration for that? This way we could enable any number of breakpoints while maintaining backwards compatibility.

**Comment by operatorone** [comment]
This is a great idea. In projects, it differs a lot what the recommendations for responsiveness are and with that way, you can match every need.

**Comment by operatorone** [comment]
Any news on this?

**Comment by operatorone** [comment]
Still no news? 😉

**Comment by rabauss** [comment]
@ausi obviously, in v3 there are now the following columns implemented: `xlarge`, `large`, `medium`, `small`, `xsmall` - so two more breakpoints than in v2 👍 

what's your idea of a bundle config? would you add a config for reusable named column sets which you can pick/select in content elements and with #4 maybe also in forms?

**Comment by ausi** [comment]
> what's your idea of a bundle config?

I thought of configuring the number of breakpoints (and maybe the corresponding `max-width`). The problem with a bundle config is, that you cannot have different settings per theme. So I’m not sure what the best solution is here. Mabe just adding `xlarge` and `xsmall` is the more feasible solution.

**Comment by operatorone** [comment]
I think it's the best way just to have at least these 5 inputs for columns and don't offer a flexible configuration. I asume you can cover a very big share of needs with that method already.
